<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Learn</class>
 <widget class="QFrame" name="Learn">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(60, 70, 35);</string>
  </property>
  <widget class="QLabel" name="labelNormal">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>查看原文和译文：</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTitle">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>751</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>谏太宗十思疏</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelSplitRead">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>260</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>逐分句阅读模式：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonNormal">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>160</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(117, 133, 67);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>查看原文和译文</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonExit">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>635</y>
     <width>140</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(117, 133, 67);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>返回首页</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelAuthor">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>作者：魏征</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSplitRead">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>260</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(117, 133, 67);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>逐分句阅读模式</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelWord">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>370</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>加点字查看模式：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonWord">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>370</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(117, 133, 67);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>加点字查看模式</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelSent">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>480</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>重点句查看模式：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSent">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>480</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(117, 133, 67);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>重点句查看模式</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonChoose">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>580</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(117, 133, 67);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>选择篇目</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelChoose">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>580</y>
     <width>741</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>未选择篇目！</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
